### DISCUSSION TITLE
Session for login

### DISCUSSION THREAD

**Discussion by MaheshkumarSundaram** [bpmnServer/bpmn-server#176] [General]
Hi @ralphhanna,

I suppose there is no session for the login when security is enabled as I see it doesn't expire anytime. Would it possible to add a session to force login again like once a day or something?

Also, what is the difference between `REQUIRE_AUTHENTICATION` and `ENFORCE_SECURITY`? What if once is `true` while the other is `false`. 

Thanks

**Comment by ralphhanna** [comment]
Sorry, I have missed this one,
### session time out:
![image](https://github.com/bpmnServer/bpmn-server/assets/11893416/e33bfad8-782a-47ab-82ae-0f8a8a75c5e8)

### REQUIRE_AUTHENTICATION
is for front-end to require users to login before using the app
### ENFORCE_SECURITY
is for back-end to ensure the passed-on user is authorized to perform the function

Sorry, I am working on better docs

Thanks
